src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        for(int i = 0; i < 3; i++) {
```

Comment:
Consider replacing the magic number `3` with a constant. This will make your code more readable and easier to maintain. For example, you could define a constant like `private static final int NUMBER_OF_BALLS = 3;` and use it in the loop condition.